contrib/jmx-metrics/src/main/groovy/io/opentelemetry/contrib/jmxmetrics/OtelHelper.groovy

```diff
@@ -23,11 +23,10 @@ import io.opentelemetry.metrics.LongCounter
 import io.opentelemetry.metrics.LongUpDownCounter
 import io.opentelemetry.metrics.LongValueRecorder
 
-import javax.management.MBeanServerConnection
 import javax.management.ObjectName
 
 class OtelHelper {
-    private static final String SCALAR = '1'
+    protected static final String SCALAR = '1'
```

Comment:
Did you want default access?

Reply:
This was used elsewhere earlier in updates but is no longer the case.  Reverted change.